pipelines/broad/arrays/imputation/Imputation.wdl

```diff
@@ -74,8 +66,7 @@ workflow Imputation {
         input_vcfs = select_first([single_sample_vcfs]),
         input_vcf_indices = select_first([single_sample_vcf_indices]),
         output_vcf_basename = "merged_input_samples",
-        bcftools_docker = bcftools_docker_tag,
-        mem = merge_ssvcf_mem_gb
+        memory_mb = merge_ssvcf_mem_gb
```

Comment:
we are trying to use memory in mb but merge_ssvcf_mem_gb is set to 3. i'm not sure how the task handles it, but should merge_ssvcf_mem_gb be increased here?

Reply:
going to update this so merge_ssvcf_mem_gb is changed to merge_ssvcf_mem_mb in the actual wdl, and i will set that to 3000 mb instead of 3 gb, just to keep it all consistent!